Makefile

```diff
@@ -0,0 +1,9 @@
+black: 
+	black test_linters.py
+
+flake8:
+	flake8 test_linters.py
```

Comment:
You need to the configure the linter so it can check all .py files, not just one (as you did). Please fix this.